src/breadcrumb/breadcrumb.tsx

```diff
@@ -137,13 +137,36 @@ export const Breadcrumb = ({
     };
 
     const renderFade = () => {
+        let fadeColorSet: FadeColorSet;
+
+        if (Array.isArray(fadeColor) && fadeColor.length > 0) {
```

Comment:
if the check is nested like this, then there's no need to cast FadeColorSet afterwards
```
         if (Array.isArray(fadeColor)) {
            if (fadeColor.length) {
               // ...
            }
```

Reply:
Haha nested ifs are ugly. :P